README.md

```diff
@@ -32,14 +32,18 @@ Unless you have run `make install` to install the `seminator` binary and all the
 
 ## Experimental evaluation
 
-Version 1.1 of Seminator was presented in the paper _František Blahoudek, Alexandre Duret-Lutz, Mikuláš Klokočka, Mojmir Kretinsky and Jan Strejcek. **Seminator: A Tool for Semi-Determinization of Omega-Automata.** In Proceedings of [LPAR-21](http://easychair.org/smart-program/LPAR-21/LPAR-index.html), 2017.  Version 1.2 add some minor optimizations, and an updated experimental evaluation.  If you want to see those results, check out the source for [version 1.2](https://github.com/adl/seminator/tree/v1.2.0).
+Version 1.1 of Seminator was presented in the paper _František Blahoudek, Alexandre Duret-Lutz, Mikuláš Klokočka, Mojmir Kretinsky and Jan Strejcek. **Seminator: A Tool for Semi-Determinization of Omega-Automata.** In Proceedings of [LPAR-21](http://easychair.org/smart-program/LPAR-21/LPAR-index.html), 2017.  Version 1.2 adds some minor optimizations, and an updated experimental evaluation.  If you want to see those results, check out the source for [version 1.2](https://github.com/adl/seminator/tree/v1.2.0).
 
-Experimental evaluation for version 2.0 is kept separate from this repository, and can be found at *FIXME*.
+The experimental evaluation for version 2.0 is now kept is a [separate repository](https://github.com/xblahoud/seminator-evaluation).
```

Comment:
is now kept is -> is now kept **in**

Reply:
Fixed, along with a few other typos caught by ispell in this file.